packages/zero-runtime/src/keyframes.d.ts

```diff
@@ -0,0 +1,4 @@
+// @TODO - Implement correct style definitions
```

Comment:
Are you going to do it in this PR? or should this be removed?

Reply:
There will be a separate PR for typescript support